samples/Emacs Lisp/filenames/Cask

```diff
@@ -0,0 +1,8 @@
+(package "cask-example" "1.2.3" "Example package for Emacs Cask")
```

Comment:
Because sample files are used for training the Bayesian classifier and for tests, we prefer to extract them from real applications (with the occasional inconsistencies, etc.). Do you know of such a file you could use (under permissive license)?